pages/news/nightstand.md

```diff
@@ -0,0 +1,39 @@
+
+<h2>Scenario</h2>
+
+<p>A tired traveler arrives at a hotel late at night and finds there is no clock in the room.  Fortunately, this person has an AsteroidOS watch with nightstand mode.  She uses the settings to select the kind of display that she would like to use for this and enables the mode.  Then she puts the watch on its charging stand where it acts as sort of a bedside clock, prominently featuring the state of charge and the current time.  She wears glasses and does not want to have to put them on to see the time, so she chooses a watchface that is high contrast and features large numbers for the time.  Even with that, it's not always that easy to see the time, so it is configured so that at 3AM with no glasses, she can pick up the watch and bring it closer to note the time (ugh!  3AM!) and replace it on the stand within some configurable period of time with the watch not leaving nightstand mode.
+
+<h2>Requirements</h2>
+<p><em>Note:</em>Numbering is arbitrary and not meant to signify priority; just to provide a convenient handle for conversation.</p>
+<ol>
+<li>When nightstand mode is enabled, and the charger is connected (i.e. nightstand mode = on) change the display to show state of charge</li>
+<li>When nightstand mode is on, display the time in a way that's clear even without glasses</li>
+<li>The user should be able to enable/disable nightstand mode</li>
+<li>The user should be able to select/configure the nightstand display</li>
+<li>The user should be able to disconnect the charger for a brief period without changing from the nightstand display</li>
+<li>After the brief period expires, the display should revert to the regular watchface</li>
+<li>The duration of the brief period should be configurable from 0 to, say, 30 seconds</li>
+<li>When nightstand mode is on, the display should rotate based on orientation</li>
+</ol>
+
+<h2>Implementation</h2>
+<p>All of the requirements except rotation based on orientation are now implemented.</p>
+<p>After some discussion among developers, the way it works now is that if Nightstand mode is active and the watch is placed on a charger, by default, the currently selected watchface will be used for this mode.  Typically, this means that each watchface has sort of an alternative mode that adds some kind of charge indicator ring.</p>
+<p>To accomodate our glasses-wearing user as described in the scenario above, one can also select a custom watchface that will be used just when Nightstand mode is active.  This allows users to choose a complex and information-filled watchface during the day, but an easier-to-read version for nightstand mode.</p>
```

Comment:
I have the feeling that these few paragraphs can be simplified a lot to be an easier read for end-users. Even I who reviewed the code had to read them slowly and a couple of times to get the idea.

Something in the ~~rough direction of:

This feature is now fully-implemented! All watchfaces have been extended with a charge indicator ring that fits their style and which only shows up in night stand mode. By default, the same watchface that's used in day-time is also used for nightstand mode but to accomodate our glasses-wearing user, one can also select a custom watchface only used in Nighstand mode.

To support rotated chargers, we also have plans to extend Nighstand with auto-rotate in the future.

Don't feel like you have to re-use my text verbatim, it's just an example that keeps most of your ideas but, I feel, said in simpler terms ?

-----

Also if it turns out to be a very simple article in the end, the h2 titles might appear to be unnecessary ? Not sure about this yet, we'll see how that turns out

Reply:
These are good suggestions.  I'll simplify and resubmit.  Thanks!